### DISCUSSION TITLE
Shoko.Server and Shoko.CLI are different products ?!

### DISCUSSION THREAD

**Discussion by bigretromike** [ShokoAnime/ShokoServer#1160] [General]
Just a head-start, both does the same but both use different ProgramData folders to store data.

**Comment by Cazzar** [comment]
This is intentional and by design

**Reply by bigretromike** [reply]
it was strange because this is first application that does that in that way.

**Reply by Cazzar** [reply]
Not sure what you would want though, as it is by design, changing something like this now would break a lot of existing setups `Shoko.Trayservice` which starts up with the default instance name of `ShokoServer` is designed as a replacement for the ages old server UI and to be ran on windows systems, whereas `Shoko.CLI` is designed as the item to run on a headless server, for example in Docker and Linux server hosts.

Though both support the custom instancing setup that has existed for a while

**Reply by bigretromike** [reply]
Yeah, I'm aware that changing it not would break thinks like other updates did. Just wanted to leave "information" for anyone looking for it, because the typical approach to debug something (the quick and dirty way) is to use cli and see what those it spill out. It was a shock for me when not working server just stared to work without issues on cli version, until I found that it created his own database and config files in different path. 
Thanks for clearing it out for me

**Comment by revam** [comment]
can we close this now?
